src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1070,7 +1070,18 @@ public function insert_posts( $items = array() ) {
 				$event['post_parent'] = $k;
 			}
 
-			//if we should create a venue or use existing
+			// Do we have an existing venue for this event that we should preserve?
+			// @todo review: should we care about the potential for multiple venue IDs?
```

Comment:
I can do that, for sure: I'd certainly prefer to address in the future as currently we lack a venue equivalent for tribe_get_organizer_ids(), amongst other things, and time is a-pressing.